scripts/genesis-from-ledger.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
```

Comment:
I would recommend using a jinja template for this. It will make this much easier to read, and we already rely on jinja templates for generating CI configurations. http://jinja.pocoo.org/

Reply:
I think that's a bit of overkill for this small piece of code.

I'll make it easier to read, by putting the pre/post/lude into arrays, and printing those in a loop.

Reply:
I agree with @nholland94 here, but willing to let this go in as is